### PR TITLE
Feature/ios 159

### PR DESCRIPTION
**Pull request by jjoyce-bittorrent**
ios-159 - autostart on computer start
ios-208 - shortcut on the desktop

### CODE REVIEW

**Comment by gzphreak** [comment]
Setting the executionlevel to user doesn't break the bonjour silent installer?

**Comment by jjoyce-bittorrent** [comment]
in onInit() elevation is requested

File: project/Win32BuildSetup/genNsisInstaller.nsi [hunk 1]
```diff
@@ -372,6 +383,34 @@ Function .onInit
 
   !insertmacro BenchPing "install" "start"
 
+  ; Elevate with the UAC plugin. Code below derived from:
+  ; http://nsis.sourceforge.net/UAC_plug-in
+  ; The switch below is necessary to handle elevation failures
+  ; and secondary process termination on successes (and failures)
+  uac_tryagain:
+  !insertmacro UAC_RunElevated
```
Comment (jjoyce-bittorrent) [inline project/Win32BuildSetup/genNsisInstaller.nsi:391]: elevation is requested here.

Comment (gzphreak) [reply project/Win32BuildSetup/genNsisInstaller.nsi:391]: Thanks!

**Review by gzphreak — APPROVED** [review]
:+1:

**Pull request merged**